log_drains/add.go

```diff
@@ -0,0 +1,24 @@
+package log_drains
+
+import (
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/go-scalingo"
+	"gopkg.in/errgo.v1"
+)
+
+func Add(app string, params scalingo.LogDrainAddParams) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
```

Comment:
When calling `Notef`, the idea is when you can add more context in the error message. Here the context is that you are adding a log drain. Here is my suggestion:
```suggestion
		return errgo.Notef(err, "fail to get Scalingo client to add a log drain")
```

Hence when we see this error in Rollbar, we immediately understand what is going on, which code path we are in :)